test/system/src/main/java/io/pravega/test/system/framework/services/PravegaProperties.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Since it is a new class, use `2019` here.

Reply:
Sure :)